src/app/Enumerations/InnerErrorTypeEnum.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Helium.Enumerations
+{
+    public enum InnerErrorType
```

Comment:
shouldn't this enum be under a more specific namespace? CSE.Helium.Controllers at a minimum

Reply:
@bartr I could put it under controllers however it's not a controller. Would \Models\Enumerations be sufficient? or \Models?